Polish/ubranie/kieszen/gwizdek/zbadajgwizdek/zagwizdaj/zmarkerem/spodnie/spodnie.md

```diff
@@ -0,0 +1,2 @@
+Niczym Picasso, zabierasz się do rysowania. Nie wiedzieć czemu, narysował-eś/aś spodnie Alladyna zrobione z tysiąca złotych cekinów!

```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.